examples/allocation/zig/testdata/build.zig

```diff
@@ -0,0 +1,30 @@
+const std = @import("std");
+const CrossTarget = std.zig.CrossTarget;
+
+pub fn build(b: *std.build.Builder) void {
+    // Standard release options allow the person running `zig build` to select
+    // between Debug, ReleaseSafe, ReleaseFast, and ReleaseSmall.
+    const mode = b.standardReleaseOptions();
+
+    const exe = b.addExecutable("greet", "greet.zig");
+    exe.setTarget(CrossTarget{ .cpu_arch = .wasm32, .os_tag = .wasi });
+    exe.setBuildMode(mode);
+
+    exe.install();
+
+    const run_cmd = exe.run();
+    run_cmd.step.dependOn(b.getInstallStep());
+    if (b.args) |args| {
+        run_cmd.addArgs(args);
+    }
+
+    const run_step = b.step("run", "Run the app");
+    run_step.dependOn(&run_cmd.step);
+
+    const exe_tests = b.addTest("greet.zig");
+    exe_tests.setTarget(CrossTarget{ .cpu_arch = .wasm32, .os_tag = .wasi });
+    exe_tests.setBuildMode(mode);
+
+    const test_step = b.step("test", "Run unit tests");
```

Comment:
nit; Do we need to have these steps? I don’t see any tests in greet.zig

Reply:
I just used the generated template as is. Removed. https://github.com/tetratelabs/wazero/pull/741/commits/5cfab1321b74dd1e8367aad6074db7b64d6758fd